src/mudlet.cpp

```diff
@@ -2067,9 +2067,12 @@ void mudlet::slot_show_help_dialog_download()
 
 void mudlet::slot_show_about_dialog()
 {
-    auto pDlg = new dlgAboutDialog( this );
-    pDlg->raise();
-    pDlg->show();
+    if( ! mpAboutDlg ) {
+        mpAboutDlg = new dlgAboutDialog( this );
```

Comment:
I don't see it as a problem for things to be hanging around when not used, because it means they'll come up faster when used the next time.

No, what's a bigger concern is https://github.com/Mudlet/Mudlet/issues/737, but this'll probably be the first time you've heard about it so judge for yourself how big of a concern it is :)